docker/vars.opentest.sh

```diff
@@ -2,4 +2,9 @@
 
 export GPC_CONSUMER_SERVER_PORT="8080"
 export GPC_CONSUMER_GPC_GET_URL="https://messagingportal.opentest.hscic.gov.uk:19192"
-export GPC_CONSUMER_GPC_STRUCTURED_PATH="/B82617/STU3/1/gpconnect/fhir/Patient/\$gpc.getstructuredrecord"
\ No newline at end of file
+export GPC_CONSUMER_GPC_STRUCTURED_PATH="/B82617/STU3/1/gpconnect/fhir/Patient/\$gpc.getstructuredrecord"
+
+export GPC_CONSUMER_SPINE_CLIENT_CERT=""
+export GPC_CONSUMER_SPINE_CLIENT_KEY=""
+export GPC_CONSUMER_SPINE_ROOT_CA_CERT=""
+export GPC_CONSUMER_SPINE_SUB_CA_CERT=""
```

Comment:
won't SDS env variables also be needed ?

-  GPC_CONSUMER_SDS_URL
-  GPC_CONSUMER_SDS_APIKEY

Reply:
They are not being used right now in the service code.